ZeroWasteFriendly/Models/ZeroWastePlace.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ZeroWastePlace.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 21/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+enum ZeroWasteCategory: String {
+    case refuse, reduce, reuse, recycle, rot, repair
+}
+
+struct ZeroWastePlace {
+
+    let name: String
+    let description: String
+    let category: ZeroWasteCategory
+    let latitude: Double
+    let longitude: Double
```

Comment:
I always wrap long and lat in a structure, because they ALWAYS go together, everywhere.
U have UserLocation now, can u use it? What's the point of having UserLocation struct? will it change in future?
I'd make a Location struct with lat and long (so you don't have to import MapKit everywhere if you'd like to uuse Coordinates struct from MapKit